ethereum/package.json

```diff
@@ -22,7 +22,7 @@
     "test": "mkdir -p build/contracts && cp node_modules/@openzeppelin/contracts/build/contracts/* build/contracts/ && CHAIN_NAME=ethereum truffle test --network test",
     "migrate": "mkdir -p build/contracts && cp node_modules/@openzeppelin/contracts/build/contracts/* build/contracts/ && truffle migrate --to 6",
     "migrate:goerli": "mkdir -p build/contracts && cp node_modules/@openzeppelin/contracts/build/contracts/* build/contracts/ && truffle migrate --to 6 --network goerli --skip-dry-run",
-    "migrate:mainnet": "mkdir -p build/contracts && cp node_modules/@openzeppelin/contracts/build/contracts/* build/contracts/ && truffle migrate --to 6 --network mainnet --skip-dry-run",
+    "migrate:mainnet": "mkdir -p build/contracts && cp node_modules/@openzeppelin/contracts/build/contracts/* build/contracts/ && truffle migrate --to 6 --network mainnet",
```

Comment:
From the [truffle doc](https://trufflesuite.com/docs/truffle/reference/command-line-options/#migrate), we may want to perform the test migration before the real migration, so I removed the `--skip-dry-run` flag. cc @h0ngcha0